src/validations/css/css.js

```diff
@@ -1,17 +1,18 @@
+var i18n = require('i18next-client');
 var css = require('css');
 var Promise = require('es6-promise').Promise;
 
 var humanErrors = {
   "missing '}'": function() {
-    return "You have a starting { but no ending } to go with it.";
+    i18n.t("css.missing-curly")
   },
 
   "property missing ':'": function() {
-    return "Put a colon (:) between the property and the value.\nTry: color: red";
+    i18n.t("css.property-missing-colon")
   },
 
   "selector missing": function() {
-    return "Start every block of CSS with a selector, such as an element name or class name.\nTry: p {\n  color: red;\n}";
+    i18n.t("css.selector-missing")
   }
 };
```

Comment:
Having troubles rendering this errors

~~~
Couldn't find a human description for Error: undefined:1:3: missing '{'
    at error (http://127.0.0.1:8080/compiled/bundle.js:45467:15)
    at declarations (http://127.0.0.1:8080/compiled/bundle.js:45652:25)
    at rule (http://127.0.0.1:8080/compiled/bundle.js:45965:21)
    at rules (http://127.0.0.1:8080/compiled/bundle.js:45522:70)
    at stylesheet (http://127.0.0.1:8080/compiled/bundle.js:45486:21)
    at Object.module.exports [as parse] (http://127.0.0.1:8080/compiled/bundle.js:45969:20)
    at module.exports (http://127.0.0.1:8080/compiled/bundle.js:119411:20)
    at module.exports (http://127.0.0.1:8080/compiled/bundle.js:119371:23)
    at React.createClass.handleUpdate (http://127.0.0.1:8080/compiled/bundle.js:119314:5)
    at null.<anonymous> (http://127.0.0.1:8080/compiled/bundle.js:119162:18)
~~~